src/features/projects/runbooks/runs/runbookRunRepository.ts

```diff
@@ -88,4 +92,44 @@ export class RunbookRunRepository {
             RunbookRunServerTasks: mappedTasks,
         };
     }
+
+    async createGit(project: Project, command: RunGitRunbookCommand, gitRef: GitRef): Promise<RunGitRunbookResponse> {
```

Comment:
Is there a reason we've got `gitRef` as a separate parameter to the function here rather than as part of `RunGitRunbookCommand`?

Similar question with `project` I suppose.

Reply:
> Consistency with the new git methods in the RunbookRepository mainly. I feel moving the gitRef would be a bit jarring if your familiar with loading and saving a runbook.

Ah right, yeah this seems reasonable.

> As for the project, similar idea, it is a segregation of the location portion of the runbook, but since the space is already in the command, that's a bit broken, ain't it

Yeah, in the case of this create runbook run API the project id isn't even in the route at all, this was done to match the other equivalent routes in that part of the API. 

The property name here is also `ownerId`, I can't see one on the Server API itself, am I looking at the wrong spot?